sqldb/SCHEMA_UPDATES.md

```diff
@@ -0,0 +1,224 @@
+# SQL Schema Updates in LND
```

Comment:
very nice docs 🙏  hopefully much of this complexity will be reduced once we switch to `sqldbv2`